modules/service/src/main/scala/lucuma/odb/service/AsterismService.scala

```diff
@@ -89,7 +89,7 @@ trait AsterismService[F[_]] {
   def getAsterism(
     programId: Program.Id,
     observationId: Observation.Id
-  )(using NoTransaction[F]): F[List[(Target.Id, Target)]]
```

Comment:
This is now called from an area that needs to have a transaction. Since this only does a query, it seems like it shouldn't matter if there is one or not, so I removed the requirement.